### DISCUSSION TITLE
Missing gifs/images for some Fields

### DISCUSSION THREAD

**Discussion by balmacefa** [payloadcms/payload#1590] [Feature Requests & Ideas]
Some Fields have gifs/images, like:
https://payloadcms.com/docs/fields/array

But others field do not. This is very useful to quickly visualize the dashboard.

**Comment by DanRibbens** [comment]
@balmacefa, This is really good feedback. I agree that the docs could really benefit from adding more media overall.

As something we can continuously improve on, I'm going to convert this from an issue to a discussion. To anyone interested in helping, please open PRs as you see fit.

**Comment by denolfe** [comment]
I think we have the intention long-term to have _the actual_ component rendered in the documentation 👍
